cmd/influxd/run/server.go

```diff
@@ -226,9 +220,8 @@ func (s *Server) appendSnapshotterService() {
 
 // SetLogOutput sets the logger used for all messages. It must not be called
```

Comment:
nit: I know this isn't your change, but the comment is wrong.  Looks like someone changed the name of this method but didn't update the comment.